tests/python/unittest/test_ndarray.py

```diff
@@ -1548,6 +1548,15 @@ def test_ndarray_is_nan():
     np.testing.assert_equal(output.asnumpy(), expected_output.astype(int))
     # astype since numpy functions default return type is boolean array instead of int
 
+@with_seed()
+def test_ndarray_hardmax():
+    data = mxnet.test_utils.rand_ndarray([2,3,4],'default')
```

Comment:
can you get a random shape using these util functions - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/test_utils.py#L402

Reply:
you can use `rand_shape_nd`

Reply:
Done